packages/daimo-api/src/api/search.ts

```diff
@@ -29,8 +34,9 @@ export async function search(
 
 async function tryGetEnsAddr(prefix: string, vc: ViemClient) {
   if (prefix.length < 3) return null;
+  if (!prefix.includes(".")) return null;
   try {
-    const ensName = normalize(prefix.includes(".") ? prefix : prefix + ".eth");
+    const ensName = normalize(prefix);
```

Comment:
might be misreading this code but I think ATM this would not return the ENS if I type `nibnalin.et` right? only when I do the full `nibnalin.eth`?

Reply:
correct

the way it was before was broken:
- typing `ansgar` returns *both* `ansgar` and `ansgar.eth`, which confused some people
- then if you keep typing, `ansgar.` returns nothing

here's how it works now:
- you get typeahead search on Daimo names
- you can enter a full ENS name, and it'll find it
- you can enter a full address, and it'll find it

...if you want, we can typeahead-complete `.eth` as soon as the user types `.`, `.e` or `.et`

(for alt-domains like .cb.id, they'd still have to type the whole thing)

Reply:
sounds good, don’t feel strongly about the type ahead after ., feel free to merge as-is